pkg/etcd/etcd.go

```diff
@@ -495,8 +496,14 @@ func (e *ETCD) removePeer(ctx context.Context, id, address string) error {
 				return err
 			}
 			if u.Hostname() == address {
+				if e.address == address {
+					logrus.Fatalf("node has been delete from the cluster. Backup and delete ${datadir}/server/db if you like to rejoin the node")
```

Comment:
Lets use the same language as we do when we fail to startup due to the node ID being deleted from the cluster. How about:

```suggestion
					logrus.Fatalf("this node was removed from the cluster, you must backup and delete %s before rejoining", e.config.DataDir")
```